### PR TITLE
- updated documentation about support for TLS with incoming/outgoing traffic in SparkPost;

### PR DESCRIPTION
**Pull request by avikulkarni**

### CODE REVIEW

**Review by RSeabrease — APPROVED** [review]

File: support/faq/are-you-sending-emails-as-encrypted-messages.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 title: "Are you sending emails as Encrypted messages?"
-description: "Spark Post supports Transport Layer Security TLS TLS and its predecessor Secure Sockets Layer SSL are cryptographic protocols that provide security on the Internet and let you encrypt emails All incoming API calls use HTTPS and are SSL TLS encrypted Incoming SMTP offers TLS if the user choses to enable..."
+description: "SparkPost supports Transport Layer Security(TLS). TLS is a cryptographic protocol that provide security on the Internet and lets you encrypt emails during transit. All incoming API calls to SparkPost use HTTPS and are SSL/TLS encrypted. SparkPost also offers TLS support for incoming SMTP traffic if the user chooses to enable it."
```
Comment (cjarnold) [inline support/faq/are-you-sending-emails-as-encrypted-messages.md:3]: provide => provides

File: support/faq/are-you-sending-emails-as-encrypted-messages.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+SparkPost supports the use of Transport Layer Security (TLS). TLS and its predecessor, Secure Sockets Layer (SSL), are cryptographic protocols that provide security on the Internet and let you encrypt emails while they are in transit. All incoming API calls to SparkPost use HTTPS and are SSL/TLS encrypted. SparkPost also offers TLS for incoming SMTP traffic if the user chooses to enable it through the use of "STARTTLS" SMTP extension. SparkPost strongly recommends using TLS with SMTP to protect your message content, recipient information and API keys in transmission. This includes API keys and any details such as recipient email addresses and message content. See more details about using encryption with SMTP <a href="https://developers.sparkpost.com/api/smtp/#header-smtp-security">here</a>.
```
Comment (cjarnold) [inline support/faq/are-you-sending-emails-as-encrypted-messages.md:6]: use of "STARTTLS" => use of the "STARTTLS"

Comment (cjarnold) [reply support/faq/are-you-sending-emails-as-encrypted-messages.md:6]: > This includes API keys and any details such as recipient email addresses and message content.

That sentence seems redundant given the previous sentence.

File: support/tech-resources/smtp-rest-api-performance.md [hunk 3]
```diff
@@ -30,6 +30,8 @@ Follow these best practices in order to optimize SMTP injection rates (and ove
 * Use concurrent connections by having multiple batches running simultaneously - use up to 10 connections at a time.  Having multiple connections open and running allows more messages to be sent in a shorter period of time.  (For SparkPost Enterprise customers, please contact your TAM for guidance on the number of concurrent connections.)
 * Keep the TCP connections open between messages.  Closing and opening connections on each message is much less efficient and can slow performance. Best practice is to reuse connections for no more than 100 messages and then create a new connection. This allows local and remote resources to be cleaned up efficiently.
 * Ensure your SMTP client has pipelining enabled, to reduce the roundtrip delay on some SMTP commands.
+* SparkPost strongly recommends using TLS with SMTP to protect your message content, recipient information and API keys in transmission. This includes API keys and any details such as recipient email addresses and message content.
```
Comment (cjarnold) [inline support/tech-resources/smtp-rest-api-performance.md:33]: Like the comment above, the 2nd sentence here seems redundant.

File: support/faq/are-you-sending-emails-as-encrypted-messages.md (hunk 1 above)
Comment (avikulkarni) [reply support/faq/are-you-sending-emails-as-encrypted-messages.md:3]: Fixed.

File: support/faq/are-you-sending-emails-as-encrypted-messages.md (hunk 2 above)
Comment (avikulkarni) [reply support/faq/are-you-sending-emails-as-encrypted-messages.md:6]: Fixed both.

File: support/tech-resources/smtp-rest-api-performance.md (hunk 3 above)
Comment (avikulkarni) [reply support/tech-resources/smtp-rest-api-performance.md:33]: Fixed

**Pull request merged**